public/app/tasks/task-details.controller.js

```diff
@@ -0,0 +1,16 @@
+(function () {
+
+    _controller.$inject = ['$stateParams', 'Task', 'toastr'];
+
+    angular
+        .module('projectTracker')
+        .controller('taskDetailsController', _controller);
+
+    function _controller($stateParams, Task, toastr) {
+        var self = this;
+
+        Task.get({ id: $stateParams.id }, function (data) {
```

Comment:
You can move inline functions to named functions to make code more linear and to understand what controller does.